src/app/pages/HomePage/Graph/NetworkSelector/index.tsx

```diff
@@ -1,100 +1,133 @@
-import { FC, useState } from 'react'
+import { FC, ForwardedRef, forwardRef, ReactElement } from 'react'
 import { useTranslation } from 'react-i18next'
-import { useTheme } from '@mui/material/styles'
-import { useScreenSize } from '../../../../hooks/useScreensize'
-import AddIcon from '@mui/icons-material/Add'
 import Box from '@mui/material/Box'
-import Button from '@mui/material/Button'
-import IconButton from '@mui/material/IconButton'
-import RemoveIcon from '@mui/icons-material/Remove'
 import Typography from '@mui/material/Typography'
 import { styled } from '@mui/material/styles'
 import { COLORS } from '../../../../../styles/theme/colors'
 import { getNetworkNames, Network } from '../../../../../types/network'
-import Collapse from '@mui/material/Collapse'
 import { RouteUtils } from '../../../../utils/route-utils'
+import {
+  Select,
+  SelectOptionBase,
+  StyledSelectButton,
+  StyledSelectListbox,
+  StyledSelectOption,
+} from '../../../../components/Select'
+import { SelectRootSlotProps } from '@mui/base/Select'
+import ExpandLessIcon from '@mui/icons-material/ExpandLess'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { getNetworkIcons } from '../../../../utils/content'
+import { optionClasses } from '@mui/base/Option'
 
-const StyledNetworkSelector = styled(Box)(() => ({
+interface NetworkOption extends SelectOptionBase {
+  label: Network
+  value: Network
+}
+
+const StyledNetworkSelector = styled(Select<NetworkOption>)(({ theme }) => ({
   position: 'absolute',
-  bottom: 0,
+  bottom: theme.spacing(5),
   display: 'flex',
   width: '100%',
   justifyContent: 'center',
 }))
 
-const StyledBox = styled(Box)(({ theme }) => ({
+const StyledButton = styled(StyledSelectButton)(({ theme }) => ({
+  height: '44px',
+  minWidth: '200px',
   backgroundColor: theme.palette.layout.primaryBackground,
-  border: `solid 3px ${theme.palette.layout.lightBorder}`,
-  borderRadius: '45px',
-  padding: theme.spacing(3, 4),
-  display: 'inline-flex',
-  alignItems: 'center',
+  border: `2px solid ${theme.palette.layout.lightBorder}`,
+  color: theme.palette.layout.main,
+  '&:hover': {
+    backgroundColor: theme.palette.layout.primaryBackground,
```

Comment:
Did you mean to change to different bg on hover/focus?
```suggestion
  '&:hover, &:focus-visible': {
    backgroundColor: 'red',
```